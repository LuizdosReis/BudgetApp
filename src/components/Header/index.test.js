import React from 'react';
import { shallow } from 'enzyme';
import { HeaderPage } from './index';

let startLogout;
let wrapper;

beforeEach(() => {
  startLogout = jest.fn();
  wrapper = shallow(<HeaderPage startLogout={startLogout} />);
});

test('should render Header correctly', () => {
  expect(wrapper).toMatchSnapshot();
});

test('should handle onClick', () => {
  wrapper.find('button').simulate('click');
  expect(startLogout).toBeCalled();
});
