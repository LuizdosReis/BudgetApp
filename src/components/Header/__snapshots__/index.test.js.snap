// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<[object Object]>
  <[object Object]>
    Budget
  </[object Object]>
  <button
    onClick={[Function]}
    type="submit"
  >
    Log out
  </button>
</[object Object]>
`;
